simtools/simtel/simtel_runner.py

```diff
@@ -173,13 +181,25 @@ def run(self, test=False, force=False, input_file=None, run_number=None):
 
     @staticmethod
     def _simtel_failed(sys_output):
+        """Test if output is different than 0
```

Comment:
I think it tests if the simtel process ended successfully.

Reply:
I corrected it now